packages/dataviews/src/filter-summary.js

```diff
@@ -123,82 +123,58 @@ export default function FilterSummary( { filter, view, onChangeView } ) {
 									} )
 								}
 							>
-								{ element.label }
-							</DropdownMenuItem>
+								<DropdownMenuItemLabel>
+									{ element.label }
+								</DropdownMenuItemLabel>
+							</DropdownMenuRadioItemCustom>
 						);
 					} ) }
 				</DropdownMenuGroup>
 				{ filter.operators.length > 1 && (
-					<DropdownSubMenu
+					<DropdownMenu
 						trigger={
-							<DropdownSubMenuTrigger
+							<DropdownMenuItem
 								suffix={
-									<>
-										{ activeOperator === OPERATOR_IN &&
-											__( 'Is' ) }
-										{ activeOperator === OPERATOR_NOT_IN &&
-											__( 'Is not' ) }
-										<Icon icon={ chevronRightSmall } />{ ' ' }
-									</>
+									<span aria-hidden="true">
+										{ OPERATORS[ activeOperator ]?.label }
+									</span>
 								}
 							>
-								{ __( 'Conditions' ) }
-							</DropdownSubMenuTrigger>
+								<DropdownMenuItemLabel>
+									{ __( 'Conditions' ) }
+								</DropdownMenuItemLabel>
+							</DropdownMenuItem>
 						}
 					>
-						<DropdownMenuItem
-							key="in-filter"
-							role="menuitemradio"
-							aria-checked={ activeOperator === OPERATOR_IN }
-							prefix={
-								activeOperator === OPERATOR_IN && (
-									<Icon icon={ check } />
-								)
-							}
-							onSelect={ () =>
-								onChangeView( {
-									...view,
-									page: 1,
-									filters: [
-										...otherFilters,
-										{
-											field: filter.field,
-											operator: OPERATOR_IN,
-											value: filterInView?.value,
-										},
-									],
-								} )
-							}
-						>
-							{ __( 'Is' ) }
-						</DropdownMenuItem>
-						<DropdownMenuItem
-							key="not-in-filter"
-							role="menuitemradio"
-							aria-checked={ activeOperator === OPERATOR_NOT_IN }
-							prefix={
-								activeOperator === OPERATOR_NOT_IN && (
-									<Icon icon={ check } />
-								)
-							}
-							onSelect={ () =>
-								onChangeView( {
-									...view,
-									page: 1,
-									filters: [
-										...otherFilters,
-										{
-											field: filter.field,
-											operator: OPERATOR_NOT_IN,
-											value: filterInView?.value,
-										},
-									],
-								} )
-							}
-						>
-							{ __( 'Is not' ) }
-						</DropdownMenuItem>
-					</DropdownSubMenu>
+						{ Object.entries( OPERATORS ).map(
+							( [ operator, { label, key } ] ) => (
+								<DropdownMenuRadioItemCustom
+									key={ key }
+									name={ `filter-summary-${ filter.name }-conditions` }
```

Comment:
This renders as `filter-summary-Author-conditions` for the `author` filter.

Note, however, that the field/filter name could be `Author Name`, for example, which would be rendered as `filter-summary-Author Name-conditions`, so I thought it'd be best to use the ID instead (`author`) at https://github.com/WordPress/gutenberg/pull/57416